app/src/main/java/com/myapp/travelize/authentication/LoginFragment.kt

```diff
@@ -37,36 +40,52 @@ class LoginFragment : Fragment() {
         savedInstanceState: Bundle?
     ): View? {
         // Inflate the layout for this fragment
-        val view = inflater.inflate(R.layout.fragment_login, container, false)
+        return inflater.inflate(R.layout.fragment_login, container, false)
```

Comment:
although this gets rid of the boilerplate and redundancy, I prefer `val view = inflater.inflate(R.layout.fragment_login, container, false)` , just incase one needs to instantiate some view from the layout. So can you change it back to the way it was before?